### DISCUSSION TITLE
regenerator-runtime error message when bundling for older web browsers

### DISCUSSION THREAD

**Discussion by moritzlaube** [parcel-bundler/parcel#6313] [General]
When trying to bundle my webpage with parceljs and the browserslist config set to "defaults" I get the following error: 
`@parcel/core: Failed to resolve 'regenerator-runtime' from './node\\_modules/@parcel/transformer-js/lib/JSTransformer.js'`

When setting the browserslist config to `"since 2017-06"`. Everything works fine. Does it has to do with Babel? Installing the regenerator-runtime with `npm install regenerator-runtime` silences this message. But in my case, GSAP (also installed via npm) stops working then. Does anyone have an idea why that happens? Appreciate your help. 

(I am using `2.0.0-nightly.665`by the way...)

**Comment by mischnic** [comment]
> Installing the regenerator-runtime with npm install regenerator-runtime silences this message.

I would have suggested that as a workaround, but https://github.com/parcel-bundler/parcel/pull/6326 should actually fix this

**Reply by moritzlaube** [reply]
Thanks! How it worked for me: 
- installing `@babel/preset-env` as a dev-dependency
- installing `regenerator`(not `regenerator-runtime` as it has a smaller footprint)
- in `package.json` adding `"browserslist": ["defaults"]`
- adding a `.babelrc`file with the following content:
```
{
  "presets": [
      ["@babel/env"]
  ]
}
```
